### PR TITLE
New sample done

### PR DESCRIPTION
**Pull request by kalyankrishna1**
## Purpose
New sample covering Azure AD app roles

## Does this introduce a breaking change?

```
[ ] Yes
[X ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[X ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ X] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: 5-WebApp-AuthZ/5-1-Roles/README.md [hunk 1]
```diff
@@ -1,219 +1,252 @@
...
-       <td>@child.Name</td>
-       <td>@child.Value</td>
-     </tr>
-    }
-   }
-</table>
-```
+The following method is also added with the `Authorize` attribute with the name of the app role **DirectoryViewers**, that permits listing of roles and groups the signed-in user is assigned to.
+
+    ```CSharp
```
Comment (jmprieur) [inline 5-WebApp-AuthZ/5-1-Roles/README.md]: the ` ` ` are not aligned with the paragraph

File: 5-WebApp-AuthZ/5-2-Groups/README.md [hunk 2]
```diff
@@ -17,18 +17,23 @@ endpoint: AAD v2.0
...
 
 ## Scenario
 
-This sample first leverages the ASP.NET Core OpenID Connect middleware to sign in the user. On the home page it displays the various `claims` that the user's [ID Token](https://docs.microsoft.com/en-us/azure/active-directory/develop/id-tokens) contained. The ID token is used by the asp.net security middleware to build the [ClaimsPrincipal](https://docs.microsoft.com/en-us/dotnet/api/system.security.claims.claimsprincipal), accessible via HttpContext.User.
+This sample first leverages the ASP.NET Core OpenID Connect middleware to sign in the user. On the home page it displays the various `claims` that the user's [ID Token](https://docs.microsoft.com/en-us/azure/active-directory/develop/id-tokens) contained. The ID token is used by the asp.net security middleware to build the [ClaimsPrincipal](https://docs.microsoft.com/en-us/dotnet/api/system.security.claims.claimsprincipal), accessible via **HttpContext.User**.
 
-![Sign in with Azure AD](ReadmeFiles/sign-in.png)
+![Sign in with the Microsoft identity platform for developers (formerly Azure AD v2.0)](ReadmeFiles/sign-in.png)
```
Comment (jmprieur) [inline 5-WebApp-AuthZ/5-2-Groups/README.md]: I think we need to change the picture : I'll do it as I know use svgs

File: 5-WebApp-AuthZ/5-1-Roles/WebApp-OpenIDConnect-DotNet.csproj [hunk 3]
```diff
@@ -21,6 +21,7 @@
     <PackageReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="Microsoft.AspNetCore.Authentication.AzureAD.UI" Version="2.2.0" />
     <PackageReference Include="Microsoft.Graph" Version="1.14.0" />
+    <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="2.2.0" />
```
[the remark this answers is not in the document]
Comment (kalyankrishna1) [reply 5-WebApp-AuthZ/5-1-Roles/WebApp-OpenIDConnect-DotNet.csproj]: I really don't know, web search says VS.NET sometime adds it during New->class/view/Controller clicks.Removed

---
In reply to: [275670685](https://github.com/Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2/pull/93#discussion_r275670685) [](ancestors = 275670685)

**Comment by jmprieur** [comment]
LGTM, @kalyankrishna1 
thanks for adressing the PR comments.
:shipit:

**Pull request merged**